docker-compose-rule-core/src/main/java/com/palantir/docker/compose/connection/State.java

```diff
@@ -15,22 +15,21 @@
  */
 package com.palantir.docker.compose.connection;
 
-import com.google.common.base.Preconditions;
-import com.google.common.base.Strings;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
 public enum State {
-    Up, Exit;
+    DOWN, PAUSED, UNHEALTHY, HEALTHY;
```

Comment:
https://docs.docker.com/engine/reference/builder/#/healthcheck says "This status is initially starting"... does this map to `UNHEALTHY` initially?